rio_toa/toa_utils.py

```diff
@@ -141,3 +140,18 @@ def temp_rescale(arr, temp_scale):
     else:
         raise ValueError('%s is not a valid temperature scale'
                          % (temp_scale))
+
+
+def normalize_scale(rescale_factor, dtype):
+    default_scales = {
+        'uint8': 215,
+        'uint16': 55000,
+        'float32': 1.0}
```

Comment:
@virginiayung Should the default for the unsigned int types use the full range? (e.g. 255 instead of 215 and 65535 instead of 55000). The only advantage to using the lower values is we avoid an overflow if clip is False.